### PR TITLE
Add Dockerfile and support scripts for CoreDNS monitoring image

### PR DESCRIPTION
**Pull request by MichaelWasher**

### CODE REVIEW

File: monitor/coredns/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    iproute \
+    sysstat \
+    net-tools \ 
+    procps-ng \
+    util-linux && \
+    dnf clean all
+
+# TODO: Replace with URL from Github once public
+ADD monitor.sh /monitor/monitor.sh
+ADD collect-metrics.sh /entrypoint/collect-metrics.sh
```
Comment (sthaha) [inline monitor/coredns/Dockerfile:20]: do you want to add the entrypoint as /entrypoint/collect-metrics.sh ?

Comment (MichaelWasher) [reply monitor/coredns/Dockerfile:20]: Is this what you meant?

File: monitor/coredns/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+ENTRYPOINT trap : TERM INT; sleep infinity & /entrypoint/collect-metrics.sh
```
Comment (sthaha) [inline monitor/coredns/Dockerfile:22]: this won't work :( 

you will have to trap TERM and INT int the script itself

**Review by sthaha — APPROVED** [review]
/approve

**Pull request merged**